packages/dialtone-vue3/directives/scrollbar/scrollbar.js

```diff
@@ -7,7 +7,12 @@ export const DtScrollbarDirective = {
     OverlayScrollbars.plugin(ClickScrollPlugin);
     app.directive('dt-scrollbar', {
       mounted (el, binding) {
-        OverlayScrollbars(el, {
+        OverlayScrollbars({
+          target: el,
+          elements: {
+            viewport: el.children[0],
```

Comment:
Can we always assume `children[0]`? Is it a requirement to only have one child element or could there be more?

Reply:
It won't work if there are not children, because there will be no viewport. If there are more than one children it would also break, because it would take the first only as the scrollable element. I can add that to the doc.

Reply:
Yeah thats a limitation we should highlight if it is the case. Thanks!